packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -54,7 +54,11 @@ abstract class RunCommandBase extends FlutterCommand with DeviceBasedDevelopment
       )
       ..addFlag('purge-persistent-cache',
         negatable: false,
-        help: 'Remove all existing persistent cache.',
+        help: 'Remove all existing persistent cache. The persistent cache is '
```

Comment:
How about something like

'''
Removes all existing persistent caches. This allows reproducing shader compilation jank that
normally only happens the first time an app is run, or for reliable testing of compilation jank fixes (e.g. shader warm-up).
'''